scipy/stats/stats.py

```diff
@@ -4840,13 +5052,19 @@ def ks_2samp(data1, data2):
     that, like in the case of the one-sample K-S test, the distribution is
     assumed to be continuous.
 
-    This is the two-sided test, one-sided tests are not implemented.
-    The test uses the two-sided asymptotic Kolmogorov-Smirnov distribution.
+    For small enough sample sizes, the computation is exact.  For large sizes,
+    the computation uses the Kolmogorov-Smirnov distributions to compute an
+    approximate value.
 
     If the K-S statistic is small or the p-value is high, then we cannot
     reject the hypothesis that the distributions of the two samples
     are the same.
 
+    In the one-sided test, the alternative is that the empirical
```

Comment:
It is confusing to associate data1 and G and data2 and F

Reply:
I agree.  
It is this way to match the description (and arguments) of `kstest(rvs, cdf,...)`, which uses `G` as the ECDF of the observed random variable, and `F` for the given distribution.
[In the 3rd PR of this K-S series, `kstest` and `ks_2samp` are unified, so that 
in `kstest(rvs, cdf,...)`, `rvs` can be either data (aka `data1`), or a callable; and `cdf` can be either a CDF or data (aka `data2`).]

Suggestions for how the doc should be written for these two functions now?

Reply:
I think if `(data1, F)` and `(data2, G)` would be ok (this ordering is more natural in my view, i.e. 1 <= 2, F <= G) or `F_1` and `F_2` (probably does not look nice)